build.gradle

```diff
@@ -10,13 +10,13 @@ repositories {
   jcenter()
 }
 
-// In this section you declare the dependencies for your production and test code
+// In this section you declare the dependencies for your production and ttt code
 dependencies {
   // The production code uses the SLF4J logging API at compile time
-  // Declare the dependency for your favourite test framework you want to use in your tests.
+  // Declare the dependency for your favourite ttt framework you want to use in your tests.
```

Comment:
You may want to get rid of all the comments. 
They don't really add anything...